graylog2-web-interface/src/components/common/Wizard.jsx

```diff
@@ -19,15 +19,6 @@ class Wizard extends React.Component {
      * Array of objects which will describe the wizard. The object must
      * contain a unique 'key' attribute, a 'title' which will be shown as step link on the left side and
      * a 'component' attribute which will hold the component which is to render for the step.
-     *
```

Comment:
This was moved because Styleguidist was trying to render it as HTML and the associated .md has all the documentation anyway